app/guide/patterns.vmd

```diff
@@ -122,7 +122,7 @@ Here, we're using a match statement to check what the user entered.
 
 In patterns, `_` matches any incoming data and discards it.
 ////
-```
+```Vale
```

Comment:
Likewise here, since Vale doesn't understand the match statement yet

Reply:
ok